templates/boxMenuItem.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import Adapt from 'core/js/adapt';
+import { classes, compile } from 'core/js/reactHelpers';
+
+export default function BoxMenuItem (props) {
+
+  const {
+    displayTitle,
+    _graphic,
+    body,
+    duration,
+    linkText,
+    _isVisited,
+    _isLocked,
+    _isComplete,
+    title,
+    _isOptional,
+    _nthChild,
+    _totalChild
+  } = props;
+
+  const _globals = Adapt.course.get('_globals');
+
+  const durationLabel = duration
+    ? _globals?._menu?._boxMenu?.durationLabel
+      ? `${_globals?._menu?._boxMenu?.durationLabel.trim()} ${duration}`
+      : duration
+    : '';
```

Comment:
```suggestion
  const durationLabel = duration
    ? [_globals?._menu?._boxMenu?.durationLabel.trim(), duration].filter(Boolean).join(' ')
    : '';
```
Only add spaces between non-null text.